src/wallet/CMakeLists.txt

```diff
@@ -149,6 +125,5 @@ if (BUILD_GUI_DEPS)
     install(TARGETS wallet_merged
         ARCHIVE DESTINATION ${lib_folder})
 
-    install(FILES ${wallet_api_headers}
-        DESTINATION include/wallet)
+  add_subdirectory(api)
```

Comment:
Maybe change the indentation from 2 spaces to 4 spaces as in the lines above for the sake of readability?